<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>778</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QWidget" name="widget" native="true">
      <property name="maximumSize">
       <size>
        <width>996</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <widget class="QWidget" name="horizontalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>1</y>
         <width>731</width>
         <height>71</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="main_buttons_and_time">
        <item>
         <widget class="QPushButton" name="previous">
          <property name="text">
           <string>❙◄</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="play">
          <property name="text">
           <string>►</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pause">
          <property name="text">
           <string>❙❙</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="stop">
          <property name="text">
           <string>⬛</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="next">
          <property name="text">
           <string>►❙</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="play_time">
          <property name="text">
           <string>0:00</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="time_slider">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickInterval">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="end_time">
          <property name="text">
           <string>0:00</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QListWidget" name="list_of_songs">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>71</y>
         <width>741</width>
         <height>651</height>
        </rect>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>251</width>
         <height>51</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="volume_layout">
        <item>
         <widget class="QLabel" name="volume_label">
          <property name="text">
           <string>                             Громкость</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="volume_slider">
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>50</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksAbove</enum>
          </property>
          <property name="tickInterval">
           <number>5</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="album_pic">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>70</y>
         <width>251</width>
         <height>251</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="audio_menu">
    <property name="title">
     <string>Аудиофайл</string>
    </property>
    <addaction name="add_track"/>
    <addaction name="add_folder"/>
    <addaction name="delete_track"/>
    <addaction name="check_text"/>
   </widget>
   <widget class="QMenu" name="playlist_menu">
    <property name="title">
     <string>Плейлист</string>
    </property>
    <addaction name="mix_playlist"/>
    <addaction name="clear_playlist"/>
   </widget>
   <addaction name="audio_menu"/>
   <addaction name="playlist_menu"/>
  </widget>
  <action name="add_track">
   <property name="text">
    <string>Добавить аудиофайл(ы)</string>
   </property>
  </action>
  <action name="delete_track">
   <property name="text">
    <string>Удалить выбранные аудиофайл(ы)</string>
   </property>
  </action>
  <action name="mix_playlist">
   <property name="text">
    <string>Перемешать плейлист</string>
   </property>
  </action>
  <action name="clear_playlist">
   <property name="text">
    <string>Очистить плейлист</string>
   </property>
  </action>
  <action name="check_info">
   <property name="text">
    <string>Просмотреть информацию о выбранной песне</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>Просмотреть текст выбранного аудиофайла</string>
   </property>
  </action>
  <action name="add_folder">
   <property name="text">
    <string>Добавить каталог</string>
   </property>
  </action>
  <action name="check_text">
   <property name="text">
    <string>Просмотреть текст выбранной песни</string>
   </property>
  </action>
  <action name="download">
   <property name="text">
    <string>Скачать музыку с Youtube и добавить в плейлист</string>
   </property>
  </action>
  <action name="action_youtube">
   <property name="text">
    <string>Скачать аудиофайл с Yotube и добавить в плейлист</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Онлайн аудиоплеер</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
